src/commands/agent/generate/template.ts

```diff
@@ -31,32 +62,132 @@ export default class AgentGenerateTemplate extends SfCommand<AgentGenerateTempla
       summary: messages.getMessage('flags.agent-api-name.summary'),
       required: true,
     }),
-    'output-dir': Flags.directory({
-      char: 'd',
-      exists: true,
-      summary: messages.getMessage('flags.output-dir.summary'),
+    'agent-version': Flags.integer({
+      summary: messages.getMessage('flags.agent-version.summary'),
+      required: true,
```

Comment:
Is this really required?  We can't assume the latest version?

Reply:
Vivek wanted it explicit. We could assume the highest or prompt if we wanted to in the future

Reply:
@iowillhoit - Could you confirm the purpose of the `*.botVersion-meta.xml` files with Nelson?

I could have sworn there was a way to save different bot versions from the UI, but I can't seem to find that anymore.  Maybe the `v1.botVersion-meta.xml` files don't mean what I thought they did.

Reply:
Yep, I can confirm with Nelson. I know that we _do_ need them, we pull info from that file for the `BotTemplate`.